discovery-provider/src/queries/get_tips.py

```diff
@@ -21,34 +20,24 @@ class GetTipsArgs(TypedDict):
     limit: int
     offset: int
     user_id: int
-    receiver_min_followers: int
-    receiver_is_verified: bool
-    current_user_follows: str
-    unique_by: str
     min_slot: int
-    max_slot: int
-    tx_signatures: List[int]
 
 
 class TipResult(TypedDict):
     amount: int
-    sender: str
-    receiver: str
+    sender: User
+    receiver: User
     slot: int
     created_at: datetime
-    followee_supporters: List[str]
+    followee_supporters: List[int]
     tx_signature: str
 
 
 # Example of query with inputs:
```

Comment:
pls doublecheck that I removed all the parts of this query that should be removed